CMakeLists.txt

```diff
@@ -50,11 +55,11 @@ project(ccl VERSION 0.2.1)
     include(BuildAngpow)
 
     # Compilation flags
-    if ("${CMAKE_C_COMPILER_ID}" MATCHES "^(Apple)?Clang$")
-      # using Clang, disabling OpenMP support
-      set(CMAKE_C_FLAGS "-O3 -fomit-frame-pointer -fno-common -fPIC -std=gnu99 -DHAVE_ANGPOW -D__CCL_DATA_DIR__=${CMAKE_INSTALL_PREFIX}/share/ccl")
-    else()
-      set(CMAKE_C_FLAGS "-O3 -fomit-frame-pointer -fno-common -fPIC -std=gnu99 -fopenmp -DHAVE_ANGPOW -D__CCL_DATA_DIR__=${CMAKE_INSTALL_PREFIX}/share/ccl")
+    set(CMAKE_C_FLAGS_RELEASE "-O3 -fomit-frame-pointer -fno-common -fPIC -std=gnu99 -DHAVE_ANGPOW -D__CCL_DATA_DIR__=${CMAKE_INSTALL_PREFIX}/share/ccl")
+    set(CMAKE_C_FLAGS_DEBUG   "-Og -fomit-frame-pointer -fno-common -fPIC -std=gnu99 -DHAVE_ANGPOW -D__CCL_DATA_DIR__=${CMAKE_INSTALL_PREFIX}/share/ccl")
+    if (NOT "${CMAKE_C_COMPILER_ID}" MATCHES "^(Apple)?Clang$")
+      # When not using Clang and in Release mode, enabling OpenMP support
+      set(CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE} -fopenmp")
```

Comment:
Looks like we will need -fopenmp in debug mode as well. Possibily because angpow depends on openmp anyways.